### PR TITLE
Added support for the --server option

### LINKED ISSUE #165: Allow passing empty --browsers option to `gulp test`
Currently, it doesn't seem possible to just start the Karma server without launching any browser. This is inconvenient if I want to run test on a browser which I want to launch myself (e.g. on a VM).

I don't know, though, how to tell `gulp test` to not start any browser. `gulp test --browser=""`? :D Or `gulp test --server`?

### PR DESCRIPTION
**Pull request by Reinmar**
Feature: Added support for the `--server` option to the automated tests runner. Closes #165.

### CODE REVIEW

File: packages/ckeditor5-dev-tests/lib/utils/automated-tests/getkarmaconfig.js [hunk 1]
```diff
@@ -174,3 +158,22 @@ module.exports = function getKarmaConfig( options ) {
...
 };
 
+// Returns the value of Karma's browser option.
+// @returns {Array|null}
+function getBrowsers( options ) {
+	if ( process.env.TRAVIS ) {
+		return [ 'CHROME_TRAVIS_CI' ];
+	}
+
+	if ( options.server || !options.browsers ) {
```
Comment (ma2ciek) [inline packages/ckeditor5-dev-tests/lib/utils/automated-tests/getkarmaconfig.js]: I see, that you handled both `--server` and `--browser=''` :)

Comment (Reinmar) [inline packages/ckeditor5-dev-tests/lib/utils/automated-tests/getkarmaconfig.js]: I just wanted to keep this: https://github.com/ckeditor/ckeditor5-dev/pull/166/files#diff-31df1471c2e1fbb9040249aaaf4d952aL47, but I changed the behaviour a bit during refactoring. That wasn't intentional, but it seems that I, indeed, handled `--browsers=''` :D

**Comment by ma2ciek** [comment]
As I made a quick research I found the `--no-brosers` option in the [docs](http://karma-runner.github.io/1.0/config/configuration-file.html) to Karma`s command line. IMO it sounds better and we'll keep the convention in the option names.

**Comment by Reinmar** [comment]
Hm... with `--no-browsers` we'd have to also use `-w` (starting Karma in single run mode without browsers doesn't seem particularly useful).

**Comment by ma2ciek** [comment]
Yes, we'd have to. So maybe we should keep the current version with `--server`

**Pull request merged**